spk/ffsync/Makefile

```diff
@@ -1,47 +1,55 @@
 SPK_NAME = ffsync
-SPK_VERS = 1.5.2
-SPK_REV = 3
+SPK_VERS = 0.14.1
+SPK_REV = 4
```

Comment:
Heads-up, as the SPK_VERS is lower than precedent it may not consider it online as the latest.
Options are:
1. deleting older version online
2. renaming the new one such as `ffsync-rs` or similar.

Reply:
Hmm, okay. This was one of the first questions I asked -- if I should be making a new package. I guess it makes sense to do so. Will see what I can do.

Reply:
That can be left for later to decide once its in a working state.